projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-update-post-endpoint.php

```diff
@@ -761,7 +761,10 @@ public function write_post( $path, $blog_id, $post_id ) {
 
 				$meta = (object) $meta;
 
-				if ( Jetpack_SEO_Posts::DESCRIPTION_META_KEY === $meta->key && ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+				if (
+					( Jetpack_SEO_Posts::DESCRIPTION_META_KEY === $meta->key || Jetpack_SEO_Posts::HTML_TITLE_META_KEY === $meta->key ) &&
```

Comment:
just a minor thing, but probably `in_array( $meta-key, [ Jetpack_SEO_Posts::HTML_TITLE_META_KEY, Jetpack_SEO_Posts::DESCRIPTION_META_KEY ] )` could look a bit easier and manageable in the future

Reply:
Good recommendation, we can update these checks to use `in_array`